jupyterlab-dataops/Dockerfile

```diff
@@ -40,6 +38,17 @@ COPY resources/jupyter/ /etc/jupyter/
 # Expose port for Vertex AI compatibility
 EXPOSE 8080
 
+# use default cloudbeaver in non-bigquery target
+FROM base AS default
```

Comment:
In workflows I cannot see any additional target provider / matrix extended with this name, I mean right now these lines of code are unused.

Reply:
What is the purpose of the default target?

Does it mean that we want to use workbench in the gcp scenario only for communicating with 3 party databases, etc., and not bigquery?

Reply:
I've deleted default target then to leave no doubt how this should work eventually.